### DISCUSSION TITLE
2017/01/29/the-story-behind-our-new-beacon-tower

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#8] [Announcements]
# 2017/01/29/the-story-behind-our-new-beacon-tower

http://localhost:4000/2017/01/29/the-story-behind-our-new-beacon-tower

**Comment by jengalas** [comment]
Posted by Steven
May 2, 2018 at 7:29 pm
_Imported from WordPress_

This is cool stuff. My brother and I have recently purchased a plot of land that has one of these towers on the top of the mountain. It has IDC badges. And serial number. There is also another about 1 miles away on the next mountain top. Both are in decent shape but no beacon on top. We climbed them growing up as kids.

The deed shows that land was leased to the civil aeronautics authority. No copy of lease available where they for a specific time period? 10, 20 years

**Comment by jengalas** [comment]
Posted by Ed Engelman
October 24, 2017 at 11:26 am
_Imported from WordPress_

Thank you for your interesting and informative web page. I am involved with the planning for the restoration of the Rock Rift Fire Tower in the Town of Tompkins, NY. It is also an International Derrick Company tower. I recently visited the Gomer Hill Fire Tower in Lewis County, NY on Gomer Hill Road. It is an IDECO Fire Tower that is on a site with four FAA towers that appear to be IDECO towers as well.

**Reply by jengalas** [reply]
Posted by Harvey Hartman
October 24, 2017 at 9:17 pm
_Imported from WordPress_

Ed,

Don’t go by appearance to determine the builder of a particular tower. Look for a builder’s plate which is usually attached to one of the horizontal members of the bottommost tower section. After the Department Of Commerce took over the Lighted Airway System from the Post Office in 1926, tower construction specifications were developed that standardized the towers, Therefore, regardless of who made them under contract for the government, they all looked the same under the DoC. However, while that implies that the parts should be interchangeable, it’s not always the case.

Good Luck with your project!

Harvey
